MyStore/Settings.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <string>
+#include <JsonObject.h>
+
+class Settings
+{
+private:
+
+	static Settings* m_instance;

```

Comment:
I'd suggest to make type of m_instance Settings, not a pointer. This way it will be destroyed automatically on program shutdown. Also you'll need to change Instance() return type to Settings instead of pointer.